resources/assets/css/style.css

```
@@ -910,3 +910,10 @@ body[bp-layout=horizontal-overlap] .navbar-filters .navbar-nav .nav-link {
 [data-bs-theme=dark] .tox button.tox-tbtn--enabled {
     background: var(--tblr-tabler) !important;
 }
+
+/* Range */
+[data-bs-theme=dark] .form-range {
+    border: solid 0.06rem var(--tblr-primary-border-subtle); /* To make more contrast could use --tblr-tabler */
```

```suggestion
    border: solid 1px var(--tblr-primary-border-subtle); /* To make more contrast could use --tblr-tabler */
```

Like I said, we probably don't want the border to grow, 1px border should be fine in any font size. Otherwise we could end up with very large borders when user increases the browser font-size, and we only want the input elements to scale, border should be fine with 1px everytime.